packages/pma/pitop/pma/adc_base.py

```diff
@@ -38,7 +44,7 @@ def own_state(self):
         }
 
     def read(
-        self, number_of_samples=1, delay_between_samples=0.05, peak_detection=False
+        self, delay_between_samples=0.05, peak_detection=False, number_of_samples=None
```

Comment:
is there a reason for reordering these params? i think that can be considered a breaking change in python since keyword params can still be used positionally?

Reply:
my bad!